### PR TITLE
[TASK] Get rid of ext_tables.php

### PR DESCRIPTION
**Pull request by NamelessCoder**
Switch to TCA/Overrides where applicable, move to
ext_localconf.php where appropriate.

### CODE REVIEW

File: Configuration/TCA/Overrides/pages.php [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+	--palette--;LLL:EXT:cms/locallang_tca.xlf:pages.palettes.miscellaneous;miscellaneous,
+	--palette--;LLL:EXT:cms/locallang_tca.xlf:pages.palettes.module;module,
+	--div--;LLL:EXT:cms/locallang_tca.xlf:pages.tabs.resources,
+	--palette--;LLL:EXT:cms/locallang_tca.xlf:pages.palettes.media;media,
+	--palette--;LLL:EXT:cms/locallang_tca.xlf:pages.palettes.storage;storage,
+	--div--;LLL:EXT:lang/locallang_tca.xlf:sys_category.tabs.category, categories';
+
+$GLOBALS['TCA']['pages']['ctrl']['typeicon_classes'][\FluidTYPO3\Fluidpages\Controller\PageControllerInterface::DOKTYPE_RAW] = 'tcarecords-pages-' . \FluidTYPO3\Fluidpages\Controller\PageControllerInterface::DOKTYPE_RAW;
+
+$doktypes = '0,1,4';
```
Comment (cedricziel) [inline Configuration/TCA/Overrides/pages.php]: Are there no constants for these?

Comment (NamelessCoder) [inline Configuration/TCA/Overrides/pages.php]: Hmm, not that I'm aware of? Perhaps in pageTSconfig but I don't think there are php constants for it.

Comment (cedricziel) [inline Configuration/TCA/Overrides/pages.php]: Alright.

**Comment by cedricziel** [comment]
Rebase me ^^

**Pull request merged**